package.json

```diff
@@ -65,6 +65,7 @@
     "vite": "^4.0.2"
   },
   "dependencies": {
+    "chokidar": "^3.5.3",
```

Comment:
It should also be possible to just use Node's `fs` for file watching, if you don't want the extra dependency of `chokidar`